src/librbd/api/Mirror.cc

```diff
@@ -413,6 +413,15 @@ struct C_ImageSnapshotCreate : public Context {
 
 } // anonymous namespace
 
+template <typename I>
+const char *pool_or_namespace(I *ictx) {
```

Comment:
Since this is a local helper, move it into the anonymous namespace, before `C_ImageGetInfo` definition.